lib/ZDumpSession.c

```diff
@@ -110,12 +110,30 @@ ZLoadSession(char *buffer, int len)
 	    free(key);
 	    return (EINVAL);
 	}
+#ifdef HAVE_KRB5_CREDS_KEYBLOCK_ENCTYPE
 	ret = krb5_init_keyblock(Z_krb5_ctx, enctype, keylength, &key->keyblock);
+#else
```

Comment:
It's probably better to move the Z_krb5_init_keyblock() helper from server/kstuff.c into the library, then to replicate its contents here.

Reply:
should be done